test/e2e/virtual/apiexport/virtualworkspace_test.go

```diff
@@ -326,7 +326,23 @@ func TestAPIExportPermissionClaims(t *testing.T) {
 	bindConsumerToProviderWithPermissionClaims(ctx, consumerWorkspace, serviceProviderWorkspace, t, kcpClusterClient, cfg, identityHash)
 
 	framework.Eventually(t, func() (success bool, reason string) {
-		// Get the binding and make sure that observed permission claims are all set.
+		// Get the binding and make sure that observed export permission claims are all set.
+		binding, err := kcpClusterClient.ApisV1alpha1().APIBindings().Get(logicalcluster.WithCluster(ctx, consumerWorkspace), "cowboys", metav1.GetOptions{})
+		if err != nil {
+			return false, err.Error()
+		}
+
+		for _, claim := range binding.Status.ExportPermissionClaims {
+			if claim.IdentityHash == identityHash {
```

Comment:
I'm pretty sure we want to verify that all the claims in this list match those in the APIExport, right?

Reply:
That's what I thought; I was mostly following the pattern in the AppliedPermissionClaims, though I think that one's incorrect too. Or at least, the comment doesn't match the code.

Reply:
Let's at least make that change here